data.json

```diff
@@ -13,6 +13,60 @@
+            "com.digiplex.game/com.digiplex.game.MainActivity",
+            "com.digiplex.game2048/com.digiplex.game2048.MainActivity",
+            "com.dt.game2048/com.dt.game2048.MainActivity",
+            "com.dupuz.game2048.yellow/com.dupuz.game2048.yellow.StartAct",
+            "com.e2048.fungame/com.d2048.mobidev.MainActivity",
+            "com.e3games.plus2048/com.e3games.plus2048.MainActivity",
+            "com.entwicklerx.m2048/com.entwicklerx.m2048.Dots2048",
+            "com.estoty.game2048/com.estoty.game2048.AppActivity",
+            "com.estoty.game2048/com.estoty.game2048.game2048",
+            "com.estoty.game2048/com.estoty.game2048.HelloWorld",
+            "com.fastsoft.game2048w/com.fastsoft.game2048w.Game2048",
+            "com.freegames.twozerofoureight2048.classic/com.tpcstld.twozerogame.MainActivity",
+            "com.funlab.puzzle2048/com.funlab.puzzle2048.MainActivity",
+            "com.funlab.super2048/com.funlab.super2048.MainActivity",
+            "com.gabrielecirulli.app2048/com.gabrielecirulli.app2048.app2048",
+            "com.gabrielecirulli.app2048/com.gabrielecirulli.app2048.MainActivity",
+            "com.game.ivza/com.game.ivza.MainActivity",
+            "com.hyperdimsoft.extreme2048/com.hyperdimsoft.extreme2048.MainActivity",
+            "com.ketchapp.play2048/com.unity3d.player.UnityPlayerActivity",
+            "com.killingtimemachine.game2048Fibonacci/com.killingtimemachine.game2048Fibonacci.Game2048",
+            "com.meltingrocks.kivy2048/org.renpy.android.PythonActivity",
+            "com.mihir.fib/com.mihir.fib.MainActivity",
+            "com.mijori.game2048/com.mijori.game2048.MainActivity",
+            "com.s2apps.game2048/com.s2apps.game2048.MainActivity",
+            "com.s2apps.game2048/com.s2apps.game2048.SplashScreen",
+            "com.scn.twok48/com.scn.twok48.MenuActivity",
+            "com.spring.bird.game2048plus/com.spring.bird.game2048plus.MainActivity",
+            "com.tpcstld.twozerogame/com.tpcstld.twozerogame.MainActivity",
+            "com.uberspot.a2048/com.uberspot.a2048.MainActivity",
+            "com.v5d2048.mobidev/com.d2048.mobidev.MainActivity",
+            "com.yotadevices.yotaphone2.game2048/com.yotadevices.yotaphone2.game2048.MainActivity",
+            "fun2048.fungame.win/com.d2048.mobidev.MainActivity",
+            "game2048.b2048game.twozerofoureight2048.game/game2048.b2048game.twozerofoureight2048.game.MainActivity",
+            "game2048.b2048game.twozerofoureight2048.game/game2048.b2048game.twozerofoureight2048.game.SplashActivity",
+            "game2048.c2048game.twentyfortyeight.twozerofoureight2048.game/game2048.c2048game.twentyfortyeight.twozerofoureight2048.game.C2048Game",
+            "in.net.shreyas.game2048/in.net.shreyas.game2048.Start",
+            "it.zerocool.game2048/it.zerocool.game2048.MainActivity",
+            "org.mania2048/org.mania2048.NumberPuzzle"
+        ],
```

Comment:
jfc everyone and their mum really did make a 2048 clone didn't they

Reply:
If you thought the 2048 clones were bad. Just wait until you see all of the Flappy Bird clones on Google Play.